### DISCUSSION TITLE
Spring packages are not cached

### DISCUSSION THREAD

**Discussion by guyhillel** [tinesoft/nxrocks#189] [Q&A] [answered]
Plugin:
 `"@nxrocks/nx-spring-boot": "^8.0.3"`

My nx.json file: 
```
{
  "extends": "nx/presets/npm.json",
  "$schema": "./node_modules/nx/schemas/nx-schema.json",
  "tasksRunnerOptions": {
    "default": {
      "runner": "nx-cloud",
      "options": {
        "cacheableOperations": [
          "build",
          "lint",
          "test",
          "e2e"
        ],
        "cacheDirectory": "/tmp/mycache",
        "accessToken": "..."
      }
    }
  },
  "plugins": [
    "@nxrocks/nx-spring-boot"
  ]
}
```

My issue is that when I run `npx nx run-many -t build` the results are not cached.

If I add the `"install"` operation into the `cacheableOperations` in nx.json, the cache works well.
Should I add this operation there? Or there is another solution?

**Comment by tinesoft** [comment, marked as answer]
Thanks for your question @guyhillel 

Adding `install` (which `build` task depends on)  to the `cacheableOperations` in `nx.json` is indeed the right move.
I have created #194  to automate that in the project.

Stay tuned and check again there!
